src/objects/DataImportBatch__c.object

```diff
@@ -44,7 +44,11 @@
     </actionOverrides>
     <actionOverrides>
         <actionName>View</actionName>
-        <type>Default</type>
+        <comment>Action override created by Lightning App Builder during activation.</comment>
+        <content>Data_Import_Batch_Record_Home</content>
+        <formFactor>Large</formFactor>
+        <skipRecordTypeSelect>false</skipRecordTypeSelect>
+        <type>Flexipage</type>
     </actionOverrides>
```

Comment:
We probably cannot push upgrade changes to action overrides for existing objects, so this would need to be documented as a manual step for customers who already have NPSP.